students/piotrowski_stanislaw/lesson_02_flow_control/lost_card.py

```diff
@@ -0,0 +1,12 @@
+# https://snakify.org/lessons/for_loop_range/problems/lost_card/
+# piotrsta
+
+number_of_cards = int(input())
+summary_value = 0
+
+for i in range(1, number_of_cards + 1):
+    summary_value += i
```

Comment:
można również skorzystać ze wzoru na sumę szeregu żeby to przyśpieszyć:
https://pl.wikipedia.org/wiki/Szereg_1_%2B_2_%2B_3_%2B_4_%2B_%E2%80%A6